### DISCUSSION TITLE
Megane R.S. DRL

### DISCUSSION THREAD

**Discussion by fortizen** [cedricp/ddt4all#1093] [Q&A]
Hello, my Megane (IV) R.S. Trophy uses the Trophy Fog lights as DRLs and i want it to use Led Lights as DRLs or both the Fogs and the Leds if possible but i can't find any Configuration for them. Can someone help me?

**Comment by AndyIreland** [comment]
I think the setting you have to enable is Scandinav lights - look in your Meganes UPC/EMM ECU for it. 
If you are talking about using your dipped main headlamps as DRL's - in those countries they have that , as soon as you turn on the engine the headlights stay on all the time regardless what the light switch is on the steering wheel stalk. 

My car is Clio Mk4 with halogen headlights so this is just used as an example:
![image](https://github.com/cedricp/ddt4all/assets/49172217/1ff6d0ac-7ad7-4402-8c49-ecd8041ea41e)
